CHANGES.md

```diff
@@ -12,6 +12,7 @@
 - Improved performance and reduced memory usage of `Event` class. [#12896](https://github.com/CesiumGS/cesium/pull/12896)
 - Fixes vertical misalignment of glyphs in labels with small fonts [#8474](https://github.com/CesiumGS/cesium/issues/8474)
 - Prevent runtime errors for certain forms of invalid PNTS files [#12872](https://github.com/CesiumGS/cesium/issues/12872)
+- Fixes precision issues with Gaussian splat tilesets where tiles have a world transform and the tileset does not [#12925](https://github.com/CesiumGS/cesium/issues/12925)
```

Comment:
@keyboardspecialist could you move this to the November section and perhaps reword a bit

```suggestion
- Fixed precision issues with Gaussian splat tilesets where the root tile does not have a world transform. [#12925](https://github.com/CesiumGS/cesium/issues/12925)
```